pages/code-review.md

```diff
@@ -14,28 +14,61 @@ channel!
 
 ## Why reviews?
 
-Code reviews are incredibly important step to the development process. Our goals
-when reviewing code are to:
+Code reviews are incredibly important step to the development process. Our
+goals when reviewing code are to:
 
 - expose bugs before they make it to production;
 - ensure a consistent quality in the codebase;
 - create an environment for sharing knowledge and developing skills;
+- cross-pollination of debugging skills when problems arise;
+- cultivation of the ability to critique one’s own code more strongly;
 - encourage open communication between the entire team.
 
+## Potential obstacles to be mindful of
```

Comment:
would it be possible to frame this in terms of solutions rather than problems? (aka - include these in "tips for a successful review" or another category that makes sense)